index.html

```diff
@@ -4,18 +4,34 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/2374882fae.js"
+      crossorigin="anonymous"
+    ></script>
 
     <title class="title">Das Frankreich Quiz für Reisefans</title>
   </head>
 
   <body>
+
     <header>
-      <h1 class="headline">Quiz App</h1>
+
+    <h1 class="headline">Wenig Zeit, viel Wissen !</h1>
+
+      <span style="font-size: 3em; color: rgb(185, 238, 104); margin-left: 40%;">
```

Comment:
Avoid inline styles where possible, and move them into CSS files. This improves the maintainability of your code and separates HTML (structure) from CSS (styling).

For example, move styles like `font-size` and `color` for icons into your CSS files under appropriate classes.
```suggestion
      <span class="icon">
```